tests/e2e/matmul/generate_e2e_matmul_tests.py

```diff
@@ -176,6 +177,26 @@ def get_test_compilation_infos(
         TileWorkgroupSizePair([16, 64, 4], [16, 2, 1]),
         TileWorkgroupSizePair([1, 128, 8], [32, 1, 1]),
     ]
+  elif compilation_info_id == CompilationInfoId.SPIRVVectorizeNVIDIA:
+    tile_workgroup_size_pairs = [
+        TileWorkgroupSizePair([[32, 128], [4, 4], [0, 0, 32]], [32, 8, 1]),
```

Comment:
Hmm, these numbers are inconsistent. Say if we have the following

```
[[wg_tile_m, wg_tile_n], [t_tile_m, t_tile_n], [0, 0, tile_k]], [x, y, z]
```
then `wg_tile_m = y * t_thie_m` and `wg_tile_n = x * t_tile_n`. So we need to fix the first tile size list in the tuple.

Reply:
I think another way to move SPIR-V side to only have one tile size like what we do for CUDA side. But that can happen separately as another patch.

Reply:
Thanks. I was not aware of that constraint, but I have added it to the test so that all future sizes will obey that constraint. Yes agreed on the single tile size for a later patch.